### ISSUE TITLE
using development mode in draw.io desktop

### ISSUE THREAD

**Issue by sapirme** [jgraph/drawio-desktop#74]
I'm trying to use the development mode but I have errors after running npm start:

![prtsc](https://user-images.githubusercontent.com/18171400/49785909-b318c980-fd2a-11e8-884c-1d27592744ac.png)

**Comment by EABangalore** [comment]
i'm facing the some problem, have you found any solution, please let me know

**Comment by moty66** [comment]
Here are some details about the issue 

```
$ node --version
v10.15.0

$ npm --version
6.6.0

$ uname -a
Darwin MBP.station 18.5.0 Darwin Kernel Version 18.5.0: Mon Mar 11 20:40:32 PDT 2019; root:xnu-4903.251.3~3/RELEASE_X86_64 x86_64

$ git rev-parse HEAD
bc268f143debb0f7ef9fe1a6d16551a3843fdcca
```

Here are the list of the file that can't be located when running in dev mode 
```
Init.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
Init.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
mxClient.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
Devel.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
ElectronApp.js:3 Uncaught ReferenceError: FeedbackDialog is not defined
```

**Comment by davidjgraph** [comment]
I've pushed fixes to get this running both in and out of dev mode.

If you're in dev mode, Electron 5 requires the hosted version of draw.io, if that's what you use, to be hosted via https. I've added notes at https://github.com/jgraph/drawio-desktop/blob/master/README.md to reflect that.

**Comment by tobiashochguertel** [comment]
Hello @davidjgraph I have to the same issue, and I don't know what I can do to get the development version of draw.io working. I started with a fresh working copy of the repository drawio-desktop and followed the instructions of the https://github.com/jgraph/drawio-desktop/blob/master/README.md

Here is a directory listing of my workingcopy and the submodules:

![image](https://user-images.githubusercontent.com/3332669/83243299-ffb1c800-a19d-11ea-9e11-21bc8c94db74.png)

I checkout mxgraph repository as an additional submodule in my workingcopy with the path-name `mxgraph2` like described in the README.MD 

I modified the npm `start` script, to see the errors also in the console:

`"start": "electron --enable-logging ./drawio/src/main/webapp",`

![image](https://user-images.githubusercontent.com/3332669/83243597-6d5df400-a19e-11ea-8266-7d9e9b472e38.png)

Here are the complete Output:

```
0 hochguertelto@de-dus-nb-toh:~/c4drawio/drawio-desktop (master *+=)
$ npm start

> draw.io@13.0.3 start /Users/hochguertelto/c4drawio/drawio-desktop
> electron --enable-logging ./drawio/src/main/webapp

[11029:0529/112056.621418:WARNING:dns_config_service_posix.cc(184)] dns_config has unhandled options!
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
createWindow {}
11:20:56.772 › Checking for update
[11031:0529/112056.828030:WARNING:dns_config_service_posix.cc(184)] dns_config has unhandled options!
[11031:0529/112056.829832:WARNING:address_sorter_posix.cc(388)] FromSockAddr failed on netmask
[11029:0529/112057.561929:INFO:CONSOLE(8)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar.js (8)
[11029:0529/112057.566860:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-ActiveDirectory.js (3)
[11029:0529/112057.567918:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Advanced.js (4)
[11029:0529/112057.569224:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AlliedTelesis.js (3)
[11029:0529/112057.571047:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Android.js (4)
[11029:0529/112057.572623:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-ArchiMate.js (4)
[11029:0529/112057.574075:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-ArchiMate3.js (4)
[11029:0529/112057.574800:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Arrows2.js (4)
[11029:0529/112057.577097:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Atlassian.js (4)
[11029:0529/112057.580846:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AWS.js (4)
[11029:0529/112057.584437:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AWS3.js (4)
[11029:0529/112057.585389:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AWS3D.js (4)
[11029:0529/112057.589749:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AWS4.js (4)
[11029:0529/112057.593541:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-AWS4b.js (4)
[11029:0529/112057.595534:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Azure.js (3)
[11029:0529/112057.596840:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Basic.js (3)
[11029:0529/112057.605687:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Bootstrap.js (4)
[11029:0529/112057.606959:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-BPMN.js (4)
[11029:0529/112057.607359:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-C4.js (3)
[11029:0529/112057.608461:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Cabinet.js (4)
[11029:0529/112057.609774:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Cisco19.js (4)
[11029:0529/112057.610835:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-CiscoSafe.js (4)
[11029:0529/112057.611938:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Citrix.js (3)
[11029:0529/112057.612946:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Cumulus.js (3)
[11029:0529/112057.613736:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-DFD.js (3)
[11029:0529/112057.614990:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-EIP.js (4)
[11029:0529/112057.617516:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Electrical.js (4)
[11029:0529/112057.618718:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-ER.js (4)
[11029:0529/112057.620989:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Floorplan.js (4)
[11029:0529/112057.621208:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Flowchart.js (3)
[11029:0529/112057.621951:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-GCP.js (4)
[11029:0529/112057.625468:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-GCP2.js (3)
[11029:0529/112057.630421:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Gmdl.js (4)
[11029:0529/112057.631935:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-IBM.js (3)
[11029:0529/112057.635508:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Infographic.js (3)
[11029:0529/112057.636425:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Ios.js (4)
[11029:0529/112057.638060:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Ios7.js (4)
[11029:0529/112057.638686:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Kubernetes.js (3)
[11029:0529/112057.639558:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-LeanMapping.js (4)
[11029:0529/112057.641994:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Mockup.js (4)
[11029:0529/112057.645218:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-MSCAE.js (4)
[11029:0529/112057.646054:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Network.js (3)
[11029:0529/112057.649336:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Office.js (4)
[11029:0529/112057.651170:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-PID.js (4)
[11029:0529/112057.652355:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Rack.js (4)
[11029:0529/112057.652864:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Sitemap.js (4)
[11029:0529/112057.656787:INFO:CONSOLE(6)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Sysml.js (6)
[11029:0529/112057.657576:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-ThreatModeling.js (3)
[11029:0529/112057.660151:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Veeam.js (3)
[11029:0529/112057.662153:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-Veeam2.js (3)
[11029:0529/112057.662614:WARNING:CONSOLE(2349)] "%cElectron Security Warning (Insecure Content-Security-Policy) font-weight: bold; This renderer process has either no Content Security
    Policy set or a policy with "unsafe-eval" enabled. This exposes users of
    this app to unnecessary security risks.

For more information and help, consult
https://electronjs.org/docs/tutorial/security.
This warning will not show up
once the app is packaged.", source: electron/js2c/sandbox_bundle.js (2349)
[11029:0529/112057.663127:INFO:CONSOLE(3)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-VVD.js (3)
[11029:0529/112057.666402:INFO:CONSOLE(4)] "Uncaught ReferenceError: Sidebar is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/sidebar/Sidebar-WebIcons.js (4)
[11029:0529/112057.667703:INFO:CONSOLE(36)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/util/mxJsCanvas.js (36)
[11029:0529/112057.669029:INFO:CONSOLE(19)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/util/mxAsyncCanvas.js (19)
[11029:0529/112057.671347:INFO:CONSOLE(65)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DrawioFile.js (65)
[11029:0529/112057.672142:INFO:CONSOLE(20)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/LocalFile.js (20)
[11029:0529/112057.672541:INFO:CONSOLE(17)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/LocalLibrary.js (17)
[11029:0529/112057.673466:INFO:CONSOLE(21)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/StorageFile.js (21)
[11029:0529/112057.674241:INFO:CONSOLE(19)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/StorageLibrary.js (19)
[11029:0529/112057.674494:INFO:CONSOLE(13)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/RemoteFile.js (13)
[11029:0529/112057.674964:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/RemoteLibrary.js (11)
[11029:0529/112057.692073:INFO:CONSOLE(1737)] "Uncaught ReferenceError: mxImage is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Dialogs.js (1737)
[11029:0529/112057.701129:INFO:CONSOLE(10)] "Uncaught ReferenceError: Editor is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Editor.js (10)
[11029:0529/112057.708647:VERBOSE1:CONSOLE(171)] "Main._createAppUI: 27.574951171875ms", source: devtools://devtools/bundled/shell.js (171)
[11029:0529/112057.717334:INFO:CONSOLE(10)] "Uncaught ReferenceError: EditorUi is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/EditorUi.js (10)
[11029:0529/112057.720412:INFO:CONSOLE(8)] "Uncaught ReferenceError: EditorUi is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DiffSync.js (8)
[11029:0529/112057.721570:INFO:CONSOLE(257)] "Uncaught ReferenceError: Editor is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Settings.js (257)
[11029:0529/112057.722459:VERBOSE1:CONSOLE(171)] "Main._showAppUI: 13.18798828125ms", source: devtools://devtools/bundled/shell.js (171)
[11029:0529/112057.723328:INFO:CONSOLE(171)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DrawioFileSync.js (171)
[11029:0529/112057.724498:INFO:CONSOLE(8)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DriveComment.js (8)
[11029:0529/112057.733216:INFO:CONSOLE(15)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DrawioClient.js (15)
[11029:0529/112057.733835:INFO:CONSOLE(29)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/UrlLibrary.js (29)
[11029:0529/112057.734252:INFO:CONSOLE(13)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DriveFile.js (13)
[11029:0529/112057.734313:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DriveLibrary.js (11)
[11029:0529/112057.734352:INFO:CONSOLE(60)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DriveClient.js (60)
[11029:0529/112057.734388:INFO:CONSOLE(13)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DropboxFile.js (13)
[11029:0529/112057.734423:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DropboxLibrary.js (11)
[11029:0529/112057.734459:INFO:CONSOLE(14)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DropboxClient.js (14)
[11029:0529/112057.734675:INFO:CONSOLE(14)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitHubFile.js (14)
[11029:0529/112057.734728:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitHubLibrary.js (11)
[11029:0529/112057.737104:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitHubClient.js (11)
[11029:0529/112057.738055:INFO:CONSOLE(13)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/OneDriveFile.js (13)
[11029:0529/112057.738446:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/OneDriveLibrary.js (11)
[11029:0529/112057.740347:INFO:CONSOLE(23)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/OneDriveClient.js (23)
[11029:0529/112057.741063:INFO:CONSOLE(14)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/TrelloFile.js (14)
[11029:0529/112057.741648:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/TrelloLibrary.js (11)
[11029:0529/112057.742773:INFO:CONSOLE(12)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/TrelloClient.js (12)
[11029:0529/112057.743347:INFO:CONSOLE(13)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitLabFile.js (13)
[11029:0529/112057.743616:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitLabLibrary.js (11)
[11029:0529/112057.745380:INFO:CONSOLE(11)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/GitLabClient.js (11)
[11029:0529/112057.752542:INFO:CONSOLE(251)] "Uncaught ReferenceError: mxClient is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/App.js (251)
[11029:0529/112057.757662:INFO:CONSOLE(8)] "Uncaught ReferenceError: mxPopupMenu is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Menus.js (8)
[11029:0529/112057.763139:VERBOSE1:CONSOLE(171)] "Main._initializeTarget: 8.6240234375ms", source: devtools://devtools/bundled/shell.js (171)
[11029:0529/112057.817393:INFO:CONSOLE(239)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Pages.js (239)
[11029:0529/112057.917300:VERBOSE1:CONSOLE(171)] "Main._lateInitialization: 5.166259765625ms", source: devtools://devtools/bundled/shell.js (171)
[11029:0529/112057.929310:INFO:CONSOLE(10)] "Uncaught ReferenceError: EditorUi is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Trees.js (10)
[11029:0529/112057.943016:INFO:CONSOLE(4)] "Uncaught ReferenceError: EditorUi is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/Minimal.js (4)
[11029:0529/112057.944704:INFO:CONSOLE(23)] "Uncaught ReferenceError: mxGuide is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DistanceGuides.js (23)
[11029:0529/112057.945952:INFO:CONSOLE(456)] "Uncaught ReferenceError: mxConstants is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/mxRuler.js (456)
[11029:0529/112057.947617:INFO:CONSOLE(5)] "Uncaught ReferenceError: Graph is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DevTools.js (5)
[11029:0529/112057.956194:INFO:CONSOLE(19)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/vsdx/mxVsdxCanvas2D.js (19)
[11029:0529/112058.007398:INFO:CONSOLE(9676)] "Uncaught ReferenceError: mxConstants is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/vsdx/importer.js (9676)
(node:11032) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(node:11032) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
[11029:0529/112058.062366:INFO:CONSOLE(3130)] "Uncaught ReferenceError: EditorUi is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/graphml/mxGraphMlCodec.js (3130)
[11029:0529/112058.063953:INFO:CONSOLE(12)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/DesktopLibrary.js (12)
[11029:0529/112058.067315:INFO:CONSOLE(4)] "Uncaught TypeError: Cannot set property 'feedbackUrl' of undefined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/diagramly/ElectronApp.js (4)
11:20:58.218 › Update for version 13.0.3 is not available (latest version: 13.0.3, downgrade is disallowed).
[11029:0529/112058.219599:INFO:CONSOLE(932)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/extensions.min.js (932)
[11029:0529/112058.302981:INFO:CONSOLE(190)] "Uncaught ReferenceError: mxStencilRegistry is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/stencils.min.js (190)
[11029:0529/112058.350367:INFO:CONSOLE(1)] "Uncaught ReferenceError: mxUtils is not defined", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/js/shapes.min.js (1)
[11029:0529/112058.351256:INFO:CONSOLE(408)] "Uncaught TypeError: App.main is not a function", source: file:///Users/hochguertelto/c4drawio/drawio-desktop/drawio/src/main/webapp/index.html?dev=1&test=1&gapi=0&db=0&od=0&gh=0&gl=0&tr=0&browser=0&picker=0&mode=device&export=https%3A%2F%2Fexp.draw.io%2FImageExport4%2Fexport (408)
[11029:0529/112058.434126:INFO:CONSOLE(239)] "(electron) 'setLayoutZoomLevelLimits' is deprecated and will be removed.", source: electron/js2c/renderer_init.js (239)
```

Screenshot of draw.io window:
![image](https://user-images.githubusercontent.com/3332669/83243819-bca42480-a19e-11ea-94ce-316306f3e50b.png)

I have runned the command `export DRAWIO_ENV=dev` before `npm start`, so the enviornment variable `DRAWIO_ENV` is setuped in my terminal with the value `dev` and then I start drawio with `npm start`.

Can you please help me?
